### PR TITLE
Create gamemodes file

### PR DESCRIPTION
**Pull request by iTrojang**

### CODE REVIEW

File: Peach/peach.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+    def fetch(self,url):
+        r = requests.get(url)
+        data = r.json()
+        return data
+
+    def getName(self,name):
+        uuid = self.fetch(url=f'https://api.mojang.com/users/profiles/minecraft/{name}')['id']
+        hyapi = self.fetch(f'https://api.hypixel.net/player?key={self.key}&uuid={uuid}')
```
Comment (ChiefChippy2) [inline Peach/peach.py:16]: don't use key in param, use "Api-key" header, see docs @ https://api.hypixel.net

Comment (iTrojang) [reply Peach/peach.py:16]: why though?

Comment (TubYoub) [reply Peach/peach.py:16]: > don't use key in param, use "Api-key" header, see docs @ https://api.hypixel.net

Why would we use "Api-key" when "key" is perfectly fine? I used this myself in my Project "HyDrp" and it works

Comment (Netwyn) [reply Peach/peach.py:16]: Headers help manage rate limiting. See the "Limits" section @ https://api.hypixel.net/

Comment (ChiefChippy2) [reply Peach/peach.py:16]: Using param works fine, but it's not documented on the api docs anymore, so it's better to just follow what the docs say.
It's basically like the name parameter for the player endpoint: hypixel says it's deprecated for years, and just changed how it works one day... let's not attempt that with key :)

**Comment by Netwyn** [comment]
Not async, and invalid names

**Pull request closed without merging**